.github/workflows/python-package.yml

```diff
@@ -66,11 +74,11 @@ jobs:
         working-directory: ./docker
         env:
           AIM_VERSION: ${{ steps.capture-version.outputs.AIM_VERSION }}
-          UPDATE_LATEST_TAG: 1
+          UPDATE_LATEST_TAG: ${{ inputs.branch || 1 }}
```

Comment:
lets specify the tag to be used instead of just a flag:
nightly for nightly dev builds
latest for latest stable

so anyone can pull what they need:
```
docker pull aim  # for latest
docker pull aim:3.6.1  # for specific stable version
docker pull aim:nightly # for latest dev
docker pull aim:3.7.0.dev20220307 # for specific dev version
```

Reply:
Yep, but I guess, it's easier to handle all that inside `create-docker-images.sh` script. Cause that script is being called from other workflows as well.